src/utils/ImportExportHelper.ts

```diff
@@ -0,0 +1,59 @@
+// Utility methods to call import/export Jahia API
+
+// http://localhost:8080/cms/export/default/export.zip?exportformat=site&live=true&users=true&path=/groups&path=/settings/mail-server-1&sitebox=digitall&exportPath=abc
+// Export
+
+import RequestOptions = Cypress.RequestOptions;
+import {JahiaServer} from '../support';
+
+const API_NAME = '/cms/export';
+
+export type ExportParameters = {
```

Comment:
Out of curiosity, any particular reasons for exporting that type which is currently only used here ?